etna/datasets/tsdataset.py

```diff
@@ -410,15 +411,41 @@ def _check_endings(self, warning=False):
             else:
                 raise ValueError("All segments should end at the same timestamp")
 
+    def _inverse_transform_target_components(self, target_components_df: pd.DataFrame, target_df: pd.DataFrame):
+        """Inverse transform target components in dataset with inverse transformed target."""
+        if self.target_components_names is not None:
```

Comment:
Do we need this if? Can't we call `self.drop_target_components()` without check?